src/BackupMetadata.cs

```diff
@@ -20,21 +17,10 @@ public bool EqualsExceptForPhysicalDeviceName(BackupMetadata x, BackupMetadata y
              x.DatabaseBackupLsn == x.DatabaseBackupLsn;

```

Comment:
Not part of the changes here but something I noticed -- Should this be `x.DatabaseBackupLsn == y.DatabaseBackupLsn` or is there a reason I'm missing for why it's written this way?

Reply:
ah, great catch!
Was a typo, I'll fix that